packages/vite/src/node/plugins/asset.ts

```diff
@@ -49,6 +49,7 @@ export const cssEntriesMap = new WeakMap<Environment, Set<string>>()
 export function registerCustomMime(): void {
   // https://github.com/lukeed/mrmime/issues/3
   mrmime.mimes['ico'] = 'image/x-icon'
+  mrmime.mimes['cur'] = 'image/x-icon'
```

Comment:
I used `image/x-icon` as this was the only mime type I found on a spec or some standards.
https://mimesniff.spec.whatwg.org/#matching-an-image-type-pattern